frontend/src/components/Gallery/Gallery.jsx

```diff
@@ -28,17 +40,9 @@ const Gallery = ({ images }) => {
                     key={`${img.id}-${idx}`}
                     tabIndex={idx}
                   >
-                    <img
-                      className="search_gallery--image"
-                      src={img.src}
-                      alt={img.title}
-                    />
-                    <button
-                      className="expand-button"
-                      onMouseDown={() => handleShow(img)}
-                    >
-                      <FontAwesomeIcon icon={faExpandAlt} inverse size="md" />
-                    </button>
+                    {/* here i used separate component called "LikeUI" */}
```

Comment:
Why not adding like button here, instead of creating new component?

Reply:
In order to make "likes" independent to that image only.